dist/s_mentions

```diff
@@ -3,6 +3,12 @@
 # Find comments related to the ticket potentially indicated by the branch name. The script never
 # fails, it only warns the caller there are comments that are potentially worth checking.
 
+# Testing notes:
+# Different users have different structures in their development trees. When making changes to this
+# script, please test a few different scenarios including. When there are multiple populated
```

Comment:
I think this full stop should be a colon.

Reply:
Thanks - fixed.